core/build.gradle

```diff
@@ -75,6 +75,7 @@ dependencies {
   integTestCompile "org.scalatest:scalatest_${scalaBaseVersion}:2.2.1"
   integTestCompile "junit:junit:4.11"
 
+    testCompile "com.github.stefanbirkner:system-rules:1.17.0"
```

Comment:
Just curious, what is that?

Reply:
this library is required for https://github.com/h2oai/sparkling-water/blob/77fe4aab020d1ebc96e991ceaeee4b1f4720137f/core/src/test/scala/org/apache/spark/h2o/NewExecutorJoinsTest.scala, especially the rule for checking system exit.